code/src/layouts/intro-banner/useIntroBanner.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useState } from "react";
+
+const useIntroBanner = (images: string[], intervalTime = 6000) => {
```

Comment:
If the hook only works with the index, why would we provide an array of images? Maybe we can change the hook to return the current image or just provide the length via props.

Reply:
Refactored to variant with providing length. Thank you!